styles.css

```diff
@@ -158,16 +159,11 @@ button {
     text-align: center;
 }
 
-#deletePaletteButton {
-    border-radius: 15px;
-    padding: 10px;
-}
-
 .modal-exit-button {
     height: 38px;
     width: 38px;
     border: none;
-    padding-left: 60%;
+    translate: 450%;
```

Comment:
neat

Reply:
yeah, had to do this because of the extra `<div>` that was added.